Makefile

```diff
@@ -150,9 +150,29 @@ update: sandstorm-$(BUILD)-fast.tar.xz
 
 fast: sandstorm-$(BUILD)-fast.tar.xz
 
-test: sandstorm-$(BUILD)-fast.tar.xz test-app.spk
+test: sandstorm-$(BUILD)-fast.tar.xz test-app.spk pack-meteor-testapp
 	tests/run-local.sh sandstorm-$(BUILD)-fast.tar.xz test-app.spk
 
+meteor-testapp-clean:
+	rm -rf tests/assets/meteor-testapp.spk meteor-spk-0.3.2 meteor-testapp/.meteor-spk
+
+pack-meteor-testapp:
+	@if [ ! -f ./meteor-spk-0.3.2/meteor-spk ]; then \
+		echo "Downloading meteor-spk into" $(PWD)"..." && \
+		curl https://dl.sandstorm.io/meteor-spk-0.3.2.tar.xz | tar Jxf -; \
+	fi; \
+	if [ ! -f ./tests/assets/meteor-testapp.spk ]; then \
+		cd meteor-testapp && \
+		echo "No meteor-testapp.spk found in sandstorm/tests/assets, packing one now..." && \
+		../meteor-spk-0.3.2/meteor-spk pack ../tests/assets/meteor-testapp.spk; \
+	elif test "$$(find meteor-testapp -name '*' -newer ./tests/assets/meteor-testapp.spk -not -path "*node_modules*" -not -path "*.meteor-spk*" -not -path "*.meteor*" | wc -l)" != "0" ; then \
```

Comment:
It looks like you're sort of re-implementing what `make` does here. Instead, you could have the make rule itself be called `tests/assets/meteor-testapp.spk`, and declare all the source files (and `meteor-spk-0.3.2/meteor-spk`) as dependencies. Make will then automatically rebuild the file whenever the dependencies change.